apps/auth/config.py

```
@@ -27,3 +27,37 @@
     print("\033[33m[WARN]\033[0m PASSWORD_MAX_LENGTH not set, using default value of 32")
 PASSWORD_MIN_LENGTH = int(os.getenv("PASSWORD_MIN_LENGTH", 8))
 PASSWORD_MAX_LENGTH = int(os.getenv("PASSWORD_MAX_LENGTH", 32))
+
+if not os.getenv("AUTH_PORT"):
+    print("\033[33m[WARN]\033[0m AUTH_PORT not set, using default value of 8005")
+AUTH_PORT = os.getenv("AUTH_PORT", 8005)
+
+if not os.getenv("ISSUER"):
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mISSUER not set, please set it in the .env file\033[0m"
+    )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)